.github/CODEOWNERS

```diff
@@ -1,2 +1,30 @@
 # Default code owners for entire repository
-*   @SimiHunjan @ochikov @petreze @svetoslav-nikol0v @agadzhalov
+*   @hashgraph/hedera-sdk @hashgraph/hedera-sdk-js-contributors @SimiHunjan
+
+#########################
+#####  Core Files  ######
+#########################
+
+# NOTE: Must be placed last to ensure enforcement over all other rules
+
+# Protection Rules for Github Configuration Files and Actions Workflows
+/.github/                                       @hashgraph/release-engineering @hashgraph/release-engineering-managers
+/.github/workflows/                             @hashgraph/release-engineering @hashgraph/release-engineering-managers @hashgraph/hedera-sdk @hashgraph/hedera-sdk-js-contributors @SimiHunjan
```

Comment:
[@hashgraph/hedera-sdk-js-contributors](https://github.com/orgs/hashgraph/teams/hedera-sdk-js-contributors) in this group there are people who are not actively working on the JS SDK right now and on every new PR they are going to be spammed with emails. 
1. I think we should drop the contributors group. 
2. Second thing is in [hedera-sdk](https://github.com/orgs/hashgraph/teams/hedera-sdk) only Robert is a member, and he is not working on the JS SDK. 
3. In my opinion we should have a group similar to `hedera-sdk-java-maintainers` but `hedera-sdk-js-maintainers` and in this group to be only the people working actively on the JS SDK right now, and only they to be receive emails for new PRs